Packs/Code42/Playbooks/playbook-Code42_Exfiltration_Playbook.yml

```diff
@@ -269,7 +271,6 @@ tasks:
       id: 08f325cd-0162-470c-8be5-4a045309b75b
       version: -1
       name: Locate CrowdStrike Host
-      description: ""
```

Comment:
Please restore the line
`description: ""`
and fill in the appropriate value.

Reply:
Apologies, should be back now.